Yoast/wordpress-seo
# Better scope the CSS override of the modal dialog overlay color.

## Summary

This PR can be summarized in the following changelog entry:

* Preserve the Gutenberg modal dialogs overlay color while providing a custom color for the Yoast SEO modal dialog

## Relevant technical choices:

* Passed a custom CSS class name to the modal dialog overlay

## Test instructions
Repeat the steps described on https://github.com/Yoast/bugreports/issues/507 and verify the Gutenberg modal dialogs overlay still have their color.

## UI changes

## Documentation


Fixes https://github.com/Yoast/bugreports/issues/507
